src/component/Input/input.css

```
@@ -0,0 +1,34 @@
+input{
```

Используй класс вместо тега для стилизации. И добавляй этот класс на соответсвующий компонент
```suggestion
.input{
```